reflex/state.py

```diff
@@ -194,6 +194,9 @@ class State(Base, ABC, extra=pydantic.Extra.allow):
     # The router data for the current page
     router: RouterData = RouterData()
 
+    # The hydrated bool.
```

Comment:
Do we have to do anything to specify that this is a `Var`? Because that logic currently lives in the `init_subclass` method

Reply:
substates will pick it up as a Var, but it wouldn't be directly usable off of `rx.State` itself, as far as i can tell.